<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 41, 109);</string>
  </property>
  <widget class="QPushButton" name="btnMainMenu">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.966, stop:0.85 #CAF2FA, stop:0.99 #656B87);
color: rgb(230, 225, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: AlignCenter;
color: rgb(71, 113, 177);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>Go Back to Main</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeBackground">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.966, stop:0.85 #CAF2FA, stop:0.99 #656B87);
color: rgb(230, 225, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: AlignCenter;
color: rgb(71, 113, 177);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="title">
    <string>Change Background color of Hangman Game</string>
   </property>
   <widget class="QRadioButton" name="radBlue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radPurple">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Purple</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBlack">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
